neptune/new/internal/init/run.py

```diff
@@ -99,102 +99,107 @@ def init_run(
+            Possible values are 'async', 'sync', 'offline', 'read-only', and 'debug'.
+        name: Editable name of the run. Defaults to 'Untitled'.
+            The name is displayed in the run details and as a column in the runs table.
+        description: Editable description of the run. Defaults to `''`.
+            The description is displayed in the run details and can be added to the runs table as a column.
+        tags: Tags of the run as a list of strings. Defaults to `[]`.
+            Tags are displayed in the run details and in the runs table as a column.
+            You can edit the tags after the run is created, either through the app or the API.
+        source_files: List of source files to be uploaded.
+            Uploaded source files are displayed in the 'Source code' tab of the run view.
+            Unix style pathname pattern expansion is supported. For example, you can pass `*.py` to upload
+            all Python files from the current directory.
+            If None is passed, the Python file from which the run was created will be uploaded.
+        capture_stdout: Whether to log the stdout of the run. Defaults to True.
+            The logged data is stored in the namespace defined by the 'monitoring_namespace' argument.
+        capture_stderr:  Whether to log the stderr of the run. Defaults to True.
+            The logged data is stored in the namespace defined by the 'monitoring_namespace' argument.
+        capture_hardware_metrics: Whether to send hardware monitoring logs (CPU, GPU, and memory utilization).
+            Defaults to True.
+            The logged data is stored in the namespace defined by the 'monitoring_namespace' argument.
+        fail_on_exception: Whether to register an uncaught exception handler to this process and,
+            in case of an exception, set the 'sys/failed' field of the run to True.
+            An exception is always logged.
+        monitoring_namespace: Namespace inside which all hardware monitoring logs are stored.
             Defaults to 'monitoring'.
-        flush_period (float, optional): In an asynchronous (default) connection mode how often asynchronous thread
-            should synchronize data with Neptune servers. Defaults to 5.
-        proxies (dict of str, optional): Argument passed to HTTP calls made via the Requests library.
-            For more information see
-            `their proxies section <https://2.python-requests.org/en/master/user/advanced/#proxies>`_.
-        capture_traceback (bool, optional):  Whether to send run’s traceback in case of an exception.
-            Defaults to `True`.
-            Tracked metadata will be stored inside `monitoring/traceback`.
+        flush_period: In the asynchronous (default) connection mode, in seconds, how often the asynchronous thread
+            should synchronize data with Neptune servers. Defaults to 5 (every 5 seconds).
```

Comment:
Potential improvement - there were recently some questions from users that understood here that they can use this to lower frequency of sending info to Neptune and save on monitoring hours and I think the description did led them to this conclusion. More info -> Prince -> Adam Nieżurawski